spec/integration/aws_elasticsearch_domain_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'spec_helper'
+
+def policy(name)
+  <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Deny",
```

Comment:
@tyler-ball Yeah, I'm not sure on the permissions.  The ID you see in the resource is actually the account ID.  I've updated the policy to be an Allow policy for the specific principal based on the ARN of the configured user, but am still seeing the failure.  AWS access policies can take over 10 minutes to apply so my guess is that we have to ensure the test user has access by default.